src/sentry/issues/escalating_issues_alg.py

```diff
@@ -0,0 +1,115 @@
+"""escalating_issues_final_alg.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1bEuyWwwUvmCKiCzo3Nh0HaWzQFnvokPI
+Validation is located at
+    https://github.com/getsentry/data-analysis/blob/main/escalating_issues_analysis/escalating_issues_validation_tool.ipynb
+"""
+
+import math
+import statistics
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Union
+
+IssueForecast = Dict[str, Union[int, str]]
+
+
+def generate_issue_forecast(
+    data: Dict[str, List[Any]], start_time: datetime
```

Comment:
Can we make this a typed dict? I think it'll make it easier to understand what's going on here